ext/README.md

```diff
@@ -405,6 +405,21 @@ Examples:
     [1,2,3,4].slice(1, 3) // return [2, 3]
     [1,2,3,4].slice(2, 4) // return [3 ,4]
 
+### Sort
+
+Sorts a list with comparable elements. If the element type is not comparable
```

Comment:
I really appreciate that you added version checks for lists. Can you make sure to indicate which version this feature was introduced in the `.md` and `.go` files?

Reply:
Done